src/Resource/Users.php

```diff
@@ -204,16 +204,25 @@ public function removeRealmRoles(string $realm, string $userId, RoleCollection $
         );
     }
 
-    public function executeActionsEmail(string $realm, string $userId): void
+    /**
+     * @param list<string>|null $actions
+     */
+    public function executeActionsEmail(string $realm, string $userId, ?array $actions = null, ?Criteria $criteria = null): void
     {
+        $payload = $actions ? [
+            'actions' => $actions,
+        ] : null;
+
```

Comment:
Hi, I was trying out the fork of this branch and it seems like there's a mistake here. The payload needs to be only the array itself (e.g. `["UPDATE_PASSWORD"]`). How should I go about contributing if I'd like to fix this and help get this merged?

Reply:
Hey, thanks for trying it out 🙂 

I changed it to
```php
    /**
     * @param list<string>|null $actions
     */
    public function executeActionsEmail(string $realm, string $userId, ?array $actions = null, ?Criteria $criteria = null): void
    {
        $this->commandExecutor->executeCommand(
            new Command(
                '/admin/realms/{realm}/users/{userId}/execute-actions-email',
                Method::PUT,
                [
                    'realm' => $realm,
                    'userId' => $userId,
                ]
                ],
                $actions,
                $criteria,
            )
        );
    }
```

Can you verify that this works now as intended?

Reply:
Yup, this works!

Reply:
Nice, thanks a lot for your help!